src/main/java/org/ecocean/servlet/export/EncounterSearchExportMetadataExcel.java

```diff
@@ -171,6 +171,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
             newEasyColumn("Occurrence.groupBehavior", columns);
             newEasyColumn("Occurrence.minGroupSizeEstimate", columns);
             newEasyColumn("Occurrence.bestGroupSizeEstimate", columns);
+            newEasyColumn("Occurrence.groupSize", columns);
```

Comment:
is there a `groupSize` on Occurrence? maybe i am missing it... i see a method for calculating an estimate one. and the above properties like `bestGroupSizeEstimate`.  could this be specific to only _some_ wildbooks?

Reply:
This ended up being a GiraffeSpotter only field. Nice catch @naknomum! I will work to convert them over to the existing, standardized field of Occurrence.bestGroupSizeEstimate

Reply:
Closing PR without merge as a result.